charts/csi-secrets-store-provider-aws/README.md

```diff
@@ -0,0 +1,42 @@
+# csi-secrets-store-provider-aws
+
+AWS Key Management Service provider for Secrets Store CSI driver allows you to get secret contents stored in AWS Key Management Service instance and use the Secrets Store CSI driver interface to mount them into Kubernetes pods.
+
+### Prerequisites
+
+- [Helm3](https://helm.sh/docs/intro/quickstart/#install-helm)
+
+### Installing the Chart
+
+- This chart installs the [secrets-store-csi-driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver) and the AWS Key Management Service provider for the driver
+
+```shell
+helm repo add csi-secrets-store-provider-aws https://to-be-defined
```

Comment:
let's make this eks-charts link

Reply:
Done. And moved here https://github.com/aws/eks-charts/pull/651